cmd/tracee-ebpf/internal/flags/flags-capture.go

```diff
@@ -73,21 +72,12 @@ func PrepareCapture(captureSlice []string) (tracee.CaptureConfig, error) {
 		} else if cap == "mem" {
 			capture.Mem = true
 		} else if strings.HasPrefix(cap, "net=") {
-			iface := strings.TrimPrefix(cap, "net=")
```

Comment:
I think it is better to keep this in a sperate line, and trim "net=" and not just "net". Anyways you then trim "=" in `PaseIface` as well.
If doing so, don't forget to update the net filter as well to trim "net="